src/main/java/net/atos/zac/authentication/UserPrincipalFilter.java

```diff
@@ -22,40 +25,56 @@
 import org.wildfly.security.http.oidc.OidcPrincipal;
 import org.wildfly.security.http.oidc.OidcSecurityContext;
 
+import net.atos.zac.zaaksturing.ZaakafhandelParameterService;
+import net.atos.zac.zaaksturing.model.ZaakafhandelParameters;
+
 @WebFilter(filterName = "UserPrincipalFilter")
 public class UserPrincipalFilter implements Filter {
 
     private static final Logger LOG = Logger.getLogger(UserPrincipalFilter.class.getName());
 
-    private static final String GROUP_MEMBERSHIP_CLAIM_NAME = "group_membership";
+    private static final String ROL_DOMEIN_ELK_ZAAKTYPE = "domein_elk_zaaktype";
```

Comment:
Een rol is iets anders. Alleen DOMEIN denk ik.

Reply:
Maar uiteindelijk is het een rol.